docs/cip/README.md

```diff
@@ -640,74 +641,34 @@ certificates specification][bls-spec].
+| Quorum size                                                       |       $\tau$        |   fraction   | Minimum fraction of committee votes required for certification                                         |                                               $\tau \geq 0.75$                                               | High threshold ensures certified EBs are known to >25% of honest nodes even with 50% adversarial stake. This widespread initial knowledge enables the network assumption that certified EBs will reach all honest parties within $L_\text{diff}$ |
+| Maximum Plutus steps per endorser block                           |          -          |  step units  | Maximum computational steps allowed for Plutus scripts in a single endorser block                      |                                                    $> 0$                                                     | Limits computational resources per EB to ensure timely validation                                                                                                                                                                                |
+| Maximum Plutus memory per endorser block                          |          -          | memory units | Maximum memory allowed for Plutus scripts in a single endorser block                                   |                                                    $> 0$                                                     | Limits memory resources per EB to ensure timely validation                                                                                                                                                                                       |
+| Maximum Plutus steps per transaction                              |          -          |  step units  | Maximum computational steps allowed for Plutus scripts in a single transaction                         |                                                    $> 0$                                                     | Limits computational resources per transaction to enable higher throughput                                                                                                                                                                       |
+| Maximum Plutus memory per transaction                             |          -          | memory units | Maximum memory allowed for Plutus scripts in a single transaction                                      |                                                    $> 0$                                                     | Limits memory resources per transaction to enable higher throughput                                                                                                                                                                              |
+
+<em>Table 2: Leios Protocol Parameters</em>
 
 </div>
 
```

Comment:
The fact that we have EB Size constraints below (in this section), but timing constraints in the protocol flow section feels very odd to me.

To me it would make most sense to have timing constraints discussed in the protocol parameter section too